src/System.Net.Sockets/src/System/Net/Sockets/UnixDomainSocketEndPoint.Unix.cs

```diff
@@ -28,108 +22,5 @@ static UnixDomainSocketEndPoint()
             Debug.Assert(s_nativePathOffset + s_nativePathLength <= s_nativeAddressSize, "Expected address size to include all of the path length");
             Debug.Assert(s_nativePathLength >= 92, "Expected max path length to be at least 92"); // per http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/sys_un.h.html
         }
-
-        public UnixDomainSocketEndPoint(string path)
```

Comment:
All of this code just moved to the other partial class, with very few changes. I'll point them out individually.